.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: "Checkout"
+      uses: actions/checkout@v3
+
+    - name: "Setup Docker Buildx"
+      uses: docker/setup-buildx-action@v2
+
+    - name: "Login to GitHub Container Registry"
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+        
+    - name: Build and Publish GitHub Actions Image
+      uses: docker/build-push-action@v4
+      with:
+        file: docker/baseimage/Dockerfile
+        push: true
+        tags: ghcr.io/${{ github.actor }}/ipa-helper-base:latest
```

Comment:
I would just name it `ipa-helper:latest`. we would have `ipa-client:latest` at some point I guess too.